### ISSUE TITLE
2-color based scheme modification

### ISSUE THREAD

**Issue by jlnrrg** [rydmike/flex_color_scheme#3]
As already outlined in #2 some schemes are too strong in difference imo.

I especially talk about the secondary value of
- "Theme Material high contrast", (lol, I assume it is the standard theme)
- "Theme Oh Mandy red" (-> if you need blue here, I would argue for one of the "Theme Brand blues" colors)
- "Amber blue theme" (-> also the blue here seems a little to prominent, a purple value or less prominent blue might be better)

I know this is all according to the individual taste, but I want to highlight the importance of non clashing color themes in the default presets. (Or name them accordingly, so the user knows that the scheme implements clashing colors)

Regarding color theory, I can recommend the [adobe color wheel](https://color.adobe.com/de/create/color-wheel), which also shows information regarding color blindness.

**Comment by rydmike** [comment]
Thanks for these observations. I don't actually disagree. Most of the built-in color schemes are fairly conservative, while a few on purpose are bold or playful, and certainly might not please all :)

Yes, the high contrast scheme exist in Flutter SDK as well, along side with the default Material guide scheme. Drill into to the source on any ColorScheme object and you will find them there in the same file. The scheme is supposed to help visually impaired, hence its name and bold design. So the Material design theme claim at least.

The "Oh Mandy Red" scheme is a play on the in the past often seen Material Red with Blue accent/FAB (or vise versa), but here with pastel color versions of the Material Red and Blue Swatch Colors. It is also almost a primary <-> secondary swap of the "Hippie Blue" scheme, both are more playful schemes, probably not everybody's cup of tea, but I kind of like them :)

The "Amber blue" and "Deep purple" where later additions, just to have two more schemes that uses colors from the standard Material swatches. Generally I'm not so fond of the standard Material swatch colors, some work well, but they are often a bit too high in saturation imo, but that is their intent, they were designed to be bold like that. I'm not very fond of the "Amber blue" Material combo either, I certainly prefer many of the more conservative schemes. Amber blue does have a design intent, and that it is to be an alternative to the Material default high contrast scheme. I think they both hurt my eyes equally much, due to their high contrasts, so I guess they serve their purpose, lol.

Ultimately FlexColorScheme is more about enabling users to create their own schemes easily. By going through all the examples it is shown how easy it is. It is totally possible and OK to use the colors in the existing schemes, and just modify one color in a scheme that does not fit with the desired design, and thus create a new scheme that way. Or as said, just make totally custom color schemes.

It is not really possibly to change any colors in the existing built-in schemes without introducing a breaking change to the package version promise. But I certainly plan to add a few more nice ones down the road. So totally new scheme suggestions are certainly welcome.

Release 1.4.0 contains some discussion and info regarding the built-in color schemes, 
[starting here](https://pub.dev/packages/flex_color_scheme#this-colorscheme-should-be-different). I thought it might be good to highlight and answer questions like this in the doc too.

Basically the built-in schemes were originally really only supposed to serve as inspiration for users to create their own schemes. I'm kind of positively surprised by the fact that many actually use them, and that they are perhaps the most popular part of the package! 😃
